lib/ansiblelint/generate_docs.py

```diff
@@ -41,3 +46,18 @@ def rules_as_rst(rules: RulesCollection) -> str:
         r += f"\n\n.. _{d.id}:\n\n{title}\n{'*' * len(title)}\n\n{d.description}"
 
     return r
+
+
+def rules_as_rich(rules: RulesCollection) -> str:
+    """Print documentation for a list of rules, returns empty string."""
+    for d in rules:
+        table = Table(show_header=True, header_style="title", box=box.MINIMAL)
+        table.add_column(d.id, style="dim", width=16)
+        table.add_column(Markdown(d.shortdesc))
+        table.add_row("description", Markdown(d.description))
+        table.add_row("version_added", d.version_added)
+        table.add_row("tags", ", ".join(d.tags))
+        table.add_row("severity", d.severity)
+        console.print(table)
```

Comment:
This looks like a hack. Can't you buffer the output and return it?

Reply:
Indeed, I asked on https://github.com/willmcgugan/rich/issues/253 to see if there is a nicer way of achieving that.